mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -1673,6 +1673,33 @@ LogicalResult ReduceOp::verify() {
   return success();
 }
 
+//===-----------------------------------------------------===//
+// BlockwiseFillOp
+//===-----------------------------------------------------===//
+
+LogicalResult BlockwiseFillOp::verify() {
+  MemRefType memrefType = getMemref().getType();
+  if (memrefType.getRank() != 1) {
```

Comment:
Nit: I think there's a way to write this constraint in the tablegen

Reply:
Noted. I ll slide this in with another PR.